ops/services/app_functions/report_stream_batched_publisher/infra/_data.tf

```diff
@@ -38,8 +38,8 @@ data "azurerm_storage_account" "app" {
 data "azurerm_storage_account_sas" "sas" {
   connection_string = data.azurerm_storage_account.app.primary_connection_string
   https_only        = true
-  start             = "2023-01-01"
-  expiry            = "2023-12-31"
+  start             = formatdate("YYYY-MM-DD", timeadd(timestamp(), "-24"))
```

Comment:
Just to clarify...is this value (24) in hours? I know we need to have a buffer; I'm just trying to figure out how much of one this gives us.

Reply:
This is 24 hours. Because this is a date and not a timestamp, giving us a full day felt like a valid way to give us an appropriate buffer.

I can test the start date in a lower environment with different values (0, 1, 12, 24) if you're ok with this approach to generating the dates.

Reply:
No need for additional tests; I trust this approach. Thanks for the clarification!